packages/sources/svelte/rollup.config.js

```diff
@@ -29,6 +29,14 @@ const components = [
     folder: 'indicators',
     name: 'VtmnBadge',
   },
+  {
+    folder: 'structure',
+    name: 'VtmnList',
+  },
+  {
+    folder: 'structure',
+    name: 'VtmnListItem',
+  },
```

Comment:
Is it possible to hide this component for the library inside the `VtmnList` component please? Because noone need to use `VtmnListItem` without a List. All the behavior could be stored in `VtmnList` component by passing `<li>` inside the slot of `<VtmnList>`. Thanks a lot.

Reply:
Hi @lauthieb , all right, you can Invite me in a google meet when you want